client/auth/token_test.go

```diff
@@ -0,0 +1,39 @@
+package auth_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/SKF/go-rest-utility/client/auth"
+)
+
+func TestRawToken_Expires_Happy(t *testing.T) {
+	t.Parallel()
+
+	token := auth.RawToken("eyJraWQiOiJ3TktkUUtMQURMdmVoRzR5V2h0RjRHZSsyUW9Rdm1DXC9vRzdFWkU0cVI2ND0iLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJiMDQ4YzAwMy1jMmJjLTRmYjAtYmVlYi00OTlmYTA2YWE5NmUiLCJldmVudF9pZCI6Ijg3ZThiMzgwLTgwODMtNDY3Yy04OWU3LTUyNzliODExMTYyNiIsInRva2VuX3VzZSI6ImFjY2VzcyIsInNjb3BlIjoiYXdzLmNvZ25pdG8uc2lnbmluLnVzZXIuYWRtaW4iLCJhdXRoX3RpbWUiOjE2MzExNjc1NTQsImlzcyI6Imh0dHBzOlwvXC9jb2duaXRvLWlkcC5ldS13ZXN0LTEuYW1hem9uYXdzLmNvbVwvZXUtd2VzdC0xX0RnN3BURGpXYyIsImV4cCI6MTYzMTY4NzAxNywiaWF0IjoxNjMxNjgzNDE3LCJqdGkiOiIzZTRmNjQ2ZC1iOGRhLTQ3OTQtYjMwNy0yYTE1MDk5MWI1ZmYiLCJjbGllbnRfaWQiOiIxOThhaXEwdXBwajBtbzZjMjh1bHZ1aWYxYiIsInVzZXJuYW1lIjoiY3VhX2VubGlnaHRAc2tmLmNvbSJ9.NA-Z_fjYI3-YnerG23He6j3ef-mhIZgQaQty46Ja_iZrGDMomW4q97uDTEjD_XY_AdILofS5_N_fCHJwhNauNgY8TGE5rmj1IE4PXYhup8PR3Mn2us7IyFlPyk537nOEqAyBIdcu42Bb_mNyG5Qt_4CuMQJnXKTOiGcEg1RXQT44OA13csfqG5lCQtje08daa91MjqOF3gg-laG1l7ETotpqhaVehjjPCeH2ii4YH4sCh9Az3sUDrmCT83UIqLxMHm22yDnwSF-IuZAOGRV_YC_KZn7NcQr9i_ah7NSiDi_GPL1jFRbQv1sytn8bflZJQ_TuyvSxe28jLkSHDVqxZQ")
+
+	exp, err := token.ParseExpires()
+	require.NoError(t, err)
+
+	require.Equal(t, time.Unix(1631687017, 0), exp)
+}
```

Comment:
no wsl here?

Reply:
Only time I've seen WSL complain about extra space is in the beginning/end or if you add two empty lines. You feel they should belong together?

Reply:
I would have expected exactly one empty line between two functions now I see 0?

Reply:
hmm, I thought you commented on line 19. But spaced it out like I think you ment.